src/CloudKit/CKCompat.cs

```diff
@@ -29,14 +29,6 @@ public CKNotificationID ()
 #endif
 
 #if XAMCORE_2_0 || !MONOMAC
-	public partial class CKFetchNotificationChangesOperation {
-
-		// `init` does not work on watchOS but we can keep compatibility with a different init
-		public CKFetchNotificationChangesOperation () : this ((CKServerChangeToken) null)
```

Comment:
@rolfbjarne Potentially (it's possible that the final 3.0 was fixed) but this fix subclassing :-|
I'll go back to see when it was fixed, I have 3.1.1 and 3.2b2 on hardware. Still have 3.0 around ? if not I'll check our database later